addons/l10n_in/models/account_invoice.py

```diff
@@ -18,7 +18,7 @@ class AccountMove(models.Model):
             ('special_economic_zone', 'Special Economic Zone'),
             ('deemed_export', 'Deemed Export')
         ], string="GST Treatment", compute="_compute_l10n_in_gst_treatment", store=True, readonly=False, copy=True)
-    l10n_in_state_id = fields.Many2one('res.country.state', string="Location of supply")
+    l10n_in_state_id = fields.Many2one('res.country.state', string="Place of supply", compute="_compute_l10n_in_state_id", store=True)
```

Comment:
So the Place of Supply is just the state of the invoicing partner?  Because should not it be readonly False if there are many complicated cases then?  And in Place of Supply I would think it is about the shipping address? (which we have in account now)

Reply:
We also think the same but this is not about the shipping address.
okay I adding read-only False

Reply:
@jva-odoo I don't see the readonly False and there would not be another way to know it is about Other Country instead of adding it as a state.  It is ugly to have a state in India that is not in India.

Reply:
we need this in EDI, and all GST reports so I suggest we not assume this.

Reply:
adding read-only False